src/types.js

```diff
@@ -61,6 +61,7 @@ export type CardEligibility = {|
   recommended?: ?boolean,
   installments?: ?boolean,
   vendors: CardVendorsEligibility,
+  guestEnabled: boolean,
```

Comment:
@wsbrunson I just checked the gql schema for this and it looks like it does allow nulls too. Should we change it to the following?

```
guestEnabled: ?boolean,
```


<img width="578" alt="Screen Shot 2022-10-26 at 4 37 18 PM" src="https://user-images.githubusercontent.com/534034/198143254-c5136743-ba24-4bb8-9837-c0ca7ae1f492.png">

Reply:
@gregjopa yeah I had actually done that but forgot to `git add --all` 🤦 
https://github.com/paypal/paypal-sdk-constants/pull/106